content/tutorials/portenta-h7/por-ard-flash/content.md

```diff
@@ -0,0 +1,256 @@
+# Using the Flash on Portenta H7 – MCU
+
+This tutorial shows you how to use the internal flash memory of the microcontroller of Portenta H7 and other MbedOS-based Arduino boards (Nano 33 BLEs, etc.). This is Part 1 of a series of projects on how to use the flash memories on the MbedOS-based Arduino boards.
```

Comment:
```suggestion
This tutorial shows you how to use the internal flash memory of the microcontroller of Portenta H7 and other MbedOS-based Arduino boards (Nano 33 BLEs, etc.). This is Part 1 of a series of projects on how to use the flash memories on the MbedOS-based Arduino boards.
```
In this tutorial we will use the internal flash memory of the microcontroller of Portenta H7 and other MbedOS-based Arduino boards (Nano 33 BLE,Nano 33 BLE Sense, etc.).

Reply:
Do we want to keep this sentence? 
This is Part 1 of a series of projects on how to use the flash memories on the MbedOS-based Arduino boards. 

Are we sure that it is gonna be a series of tutorials? and it is planned that we will release soon-ish a new tutorial related to this? If so, I would add it, if not, I would remove it

Reply:
This sketch was Part 1 in respect of https://www.arduino.cc/pro/tutorials/portenta-h7/por-ard-kvs

Reply:
End here [Portenta_Storage_Examples](https://create.arduino.cc/editor/manchuino/c15a5e66-4c54-430d-914b-9326125db6c6/preview) you have Part 3, ie. using the QSPI Flash with Partitions and FAT, LittleFS and TDBStore.